feat(guide): Collection of Custom Formats - HDR Formats - Improve description (#1760)
TRaSH-Guides/Guides: includes/cf-descriptions/dv.md

*This Custom Format will only match on* `DV` **NOT** *on* `DV HDR10` *etc*.

> Depending on the used source (WEB-DL, Bluray or Remux) it will match on the following
> 
> You will find this mainly with:

Reviewer: ```suggestion
Depending on the source (WEB-DL, Bluray, or Remux) this format will match the following:
```
